lib/Service/ImportService.php

```diff
@@ -71,9 +71,9 @@ public function __construct(
      * @param string $userId
      * @param array  $json
      *
-     * @return array|null
+     * @return \OCP\AppFramework\Db\Entity|null
      */
-    public function importArticles(string $userId, array $json)
+    public function importArticles(string $userId, array $json): ?\OCP\AppFramework\Db\Entity
```

Comment:
Why does this return a single entity?

Reply:
Because `$this->feedService->findByURLfindByURL` only returns the one (new) feed for items without a feed. If this wasn't necessary (and `$createdFeed` is still false) it return null;

Reply:
Okay, but can we use `use \OCP\AppFramework\Db\Entity;` and this then
```suggestion
    public function importArticles(string $userId, array $json): ?Entity
```